azurelinuxagent/ga/monitor.py

```diff
@@ -176,7 +176,7 @@ def init_sysinfo(self):
             self.sysinfo.append(TelemetryEventParam("RAM", ram))
             self.sysinfo.append(TelemetryEventParam("Processors", processors))
         except OSUtilError as e:
-            logger.warn("Failed to get system info: {0}", e)
+            logger.warn("Failed to get system info: {0}".format(ustr(e)))
```

Comment:
isn't logger.warn supposed to do the call to format()?

Reply:
Yes. Reverted the change (adding the ustr)